app/services/food_service.rb

```diff
@@ -0,0 +1,13 @@
+class FoodService
+  
+  def self.conn
+    conn = Faraday.new(
+      url: 'https://api.nal.usda.gov/fdc/v1/foods',
+      headers: {'X-Api-Key' => ENV['fdc_api_key']}
+    )
+  end
+  def self.search_foods(search_params)
+    resp =conn.get('search', {query: search_params})
+    JSON.parse(resp.body, symbolize_names: true)
+  end
```

Comment:
Solid, simple service. Love where you set the API key and breaking up your arguments to the Faraday connection object onto multiple lines to improve readablity/maintainability.